src/application/mLoginBox.cpp

```diff
@@ -3,16 +3,25 @@
  * All rights reserved. Distributed under the terms of the MIT license.
  */
 #include <Catalog.h>
+#include <GraphicsDefs.h>
+#include <InterfaceDefs.h>
 #include <LayoutBuilder.h>
+#include <stdlib.h>
```

Comment:
Try to use C++ headers, not their C versions. In this case, it would be `cstdlib`.